app/src/main/res/layout/activity_main.xml

```diff
@@ -33,14 +33,21 @@
             app:popupTheme="@style/ThemeOverlay.AppCompat.Light"
             android:elevation="4dp">
 
+            <de.westnordost.streetcomplete.statistics.UploadsCounter
+                android:id="@+id/uploadsCounter"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_gravity="end"
+                android:drawablePadding="2dp"
+                android:drawableStart="@drawable/ic_file_upload_24dp"
+                android:gravity="center_vertical"
+                />
```

Comment:
The component should have a proper selectable item background because it is clickable.

Reply:
I added 'android:background="?android:attr/selectableItemBackground"' to the component. Assuming that's what you had in mind.